docs/UserGuide.md

```diff
@@ -144,17 +158,10 @@ _Help window displayed after running the `help` command_
 
 Adds a patient to the patient list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION | DATE_AND_TIME | FREQUENCY]… [c/CONDITION]… [t/TAG]…`
-
-* `DATE_AND_TIME` should be in the format like 16-10-2022 1030
-* If no `DATE_AND_TIME` is provided, then a default date and time of 24 hours from creation will be set
-* `FREQUENCY` is made of two components, the number of time periods in between recurring tasks and the specific time period, e.g 3 month or 2 week
-* The time periods for `FREQUENCY` are: `days`, `weeks`, `months` and `years`
-* `FREQUENCY` can be be empty if it's a one off task, however a `DATE_AND_TIME` must be provided inorder to set a recurring task.
-* Note that capitalization does not matter for frequency.
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION | DATE TIME | INTERVAL TIME_PERIOD]… [c/CONDITION]… [t/TAG]…`
```

Comment:
I think you should add the `<>` notation here as well:
```suggestion
Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>]… [c/CONDITION]… [t/TAG]…`
```
I also think perhaps it would be good to update the parameters with the standardised format wherever possible. If I change everything after everyone is done, it skews my code contribution aka snipe your UG contribution

Reply:
```suggestion
Format: `add n/NAME p/PHONE a/ADDRESS e/EMAIL [t/TAG]… [d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>]… [c/CONDITION]… [m/MEDICATION | DOSAGE]… [r/REMARK]…`
```